Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy.swift

```diff
@@ -838,7 +838,7 @@ extension PathHierarchy.Error {
             return "Symbol links can only resolve symbols."
             
         case .lookupCollision(let partialResult, let collisions):
-            let collisionDescription = collisions.map { "Add \($0.disambiguation.singleQuoted) to refer to \($0.node.fullNameOfValue(context: context).singleQuoted)"}.sorted()
+            let collisionDescription = collisions.map { "Append '-\($0.disambiguation)' to refer to \($0.node.fullNameOfValue(context: context).singleQuoted)"}.sorted()
```

Comment:
Should we also delete the following "singleQuoted" call？

```suggestion
            let collisionDescription = collisions.map { "Append '-\($0.disambiguation)' to refer to \($0.node.fullNameOfValue(context: context))"}.sorted()
```

Reply:
I don't think so. I feel the the quotation marks makes it easier to follow where each declaration starts and ends in the error message. 

I surrounded the disambiguation interpolation with quotation marks because I also needed to prefix it with a dash and I felt that `'-\($0.disambiguation)'` was more readable than `\(("-" + $0.disambiguation).singleQuoted)`. In the rest of the code I think that we're mostly using `.singleQuoted`.